bundle/Core/Search/Elasticsearch/Query/Common/CriterionVisitor/Tags.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\TagsBundle\Core\Search\Elasticsearch\Query\Common\CriterionVisitor;
+
+use Ibexa\Contracts\Core\Persistence\Content\Type\Handler;
+use Ibexa\Contracts\Core\Repository\Values\Content\Query\Criterion;
+use Ibexa\Contracts\Elasticsearch\Query\CriterionVisitor;
+use Ibexa\Core\Search\Common\FieldNameResolver;
+
+use function array_merge;
+
+abstract class Tags implements CriterionVisitor
```

Comment:
Since we cannot obviously add Ibexa Elastic Search to requirements to silence PHPStan here, we need to ignore these errors in `phpstan.neon`

Reply:
You can just ignore every error containing the string `Ibexa\Contracts\Elasticsearch` or `Ibexa\Elasticsearch`

Reply:
fixed in a238f85b2507eb60ea393971bfebcfafe6276ebc, 2d2e4ed40563f27e9c039f920d01935797cbe360 and ebb0e601d1130be04887c2be180fb22c3e424fde